src/Http/Auth/PublicRouteWithAuthMatcher.php

```diff
@@ -0,0 +1,58 @@
+<?php
+
+declare(strict_types=1);
+
+namespace CultuurNet\UDB3\Http\Auth;
+
+use Psr\Http\Message\ServerRequestInterface;
+
+final class PublicRouteWithAuthMatcher
+{
+    private const ALWAYS = 'always';
+    private const PARAM = 'param';
```

Comment:
You can make these consts public and use them in the config instead of plain strings